### DISCUSSION TITLE
定制目录章节字体及字号

### DISCUSSION THREAD

**Discussion by ManifoldsHu** [nju-lug/NJUThesis#225] [Q&A] [answered]
在 1.1.2 版本中，使用 
`\njusetformat{chapter}{\zihao{-2}\rmfamily\centering\bfseries} % 将章标题字号设为小二 `
命令时，会同步更改目录中 chapter 的字号、字体等。

然而，在现工院的模板中，要求目录中 chapter 的字号为四号，section 及 subsection 为小四。但在正文中，章标题的字号要求为小二。二者差距过大过于明显，以至于无法通过审核。

**Comment by ManifoldsHu** [comment]
经过进一步测试，发现 v1.1.2 版本中，
`\njusetformat{chapter}{\zihao{-2}\rmfamily\centering\bfseries}` 
命令不会更改目录中 chapter 的字号 (尽管它会同步更改字体)。

但仍然请求定制目录中章标题字号大小的方法。

**Comment by atxy-blip** [comment, marked as answer]
转移到 Discussions 了。目录标题格式默认与章标题保持一致，但你也可以单独改。这里也不建议直接在格式里加带参数的 `\zihao` 命令。

```tex
\newcommand\sihao{\zihao{4}}
\newcommand\xiaoer{\zihao{-2}}
\njusetformat{chapter}{\xiaoer\rmfamily\centering\bfseries}
\njusetformat{chapterintoc}{\sihao\rmfamily\bfseries}
```

https://github.com/nju-lug/NJUThesis/blob/b3d4c24bf5927b038bef5d4066aece96c629744f/source/njuthesis.dtx#L8458-L8464

**Reply by ManifoldsHu** [reply]
经测试有效，感谢解答！

建议可以在文档中的第 5.27.4 节 (样式) 中列出可定制的样式~
